File: Class/C language/For Loop Practice/pattern5.cpp
//inverted Right half pyramid
#include<stdio.h>
main(){
	
	int num,i,j;
	
	printf("Enter Number::");
	scanf("%d", &num);
	
	printf("inverted Right half pyramid.\n");
	printf("\n");
	for(i=1;i<=num;i++){
		for(j=1;j<=num-i+1;j++){
			printf("*");
		}
		printf("\n");
	}
	
}

File: Python/class/Logical Program's/pattern5.py
num=int(input("Enter a No. of Row:-"))
for i in range(num-1):
    for j in range(num-i):
        print(" ",end="")
    for k in range(i+1):
        if k==0 or k==i:
            print("* ",end="")
        else:
            print("  ",end="")
    print()

for i in range(num):
    for j in range(i+1):
        print(" ",end="")
    for k in range(num-i):
        if k==0 or k==num-(i+1):
            print("* ",end="")
        else:
            print("  ",end="")
    print()